observability/cluster_observability_operator/ui_plugins/observability-ui-plugins-overview.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="observability-ui-plugins-overview"]
+= Observability UI plugins overview
+include::_attributes/common-attributes.adoc[]
+:context: observability-ui-plugins-overview
+
+toc::[]
+
+:FeatureName: The Cluster Observability Operator
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The {coo-first} can install and manage UI plugins to enhance the observability capabilities of the {product-title} web console.
+The plugins extend the default functionality, providing new UI features for monitoring, troubleshooting, distributed tracing, and cluster logging.
+
+== Dashboards
+
+The dashboard UI plugin supports enhanced dashboards in the {product-title} web console at *Observe* -> *Dashboards*.
+You can add other Prometheus datasources from the cluster to the default dashboards, in addition to the in-cluster datasource.
+This results in a unified observability experience across different data sources.
```

Comment:
```suggestion
You can add other Prometheus data sources from the cluster to the default dashboards, in addition to the in-cluster data source.
This results in a unified observability experience across different data sources.
```

Both [IBMSG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#data-source) and [Prometheus docs](https://grafana.com/docs/grafana/latest/datasources/prometheus/) seem to use "data source".

Reply:
done